src/ontology/external/mondo-omim-genes.robot.tsv

```diff
@@ -4048,7 +4044,6 @@ http://purl.obolibrary.org/obo/MONDO_0030887	http://identifiers.org/hgnc/6648	OM
 http://purl.obolibrary.org/obo/MONDO_0030890	http://identifiers.org/hgnc/13998	OMIM:619909	https://omim.org/entry/616741
 http://purl.obolibrary.org/obo/MONDO_0030891	http://identifiers.org/hgnc/814	OMIM:619910	https://omim.org/entry/108731
 http://purl.obolibrary.org/obo/MONDO_0030893	http://identifiers.org/hgnc/6215	OMIM:619147	https://omim.org/entry/601421
-http://purl.obolibrary.org/obo/MONDO_0030894	http://identifiers.org/hgnc/253	OMIM:619151	https://omim.org/entry/103710
```

Comment:
This should not have been removed based on https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L3

and this should be included https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L2

Reply:
fixed